pandas/core/generic.py

```diff
@@ -7708,6 +7708,8 @@ def resample(
         ----------
         rule : DateOffset, Timedelta or str
             The offset string or object representing target conversion.
+            The str value refers to frequency string.
+            The list of valid values can be found in the timeseries user guide.
```

Comment:
@anshbansal making this a link to the `timeseries user guide`, besides saying it.

Reply:
Where, pray tell - [other than on SO](https://stackoverflow.com/a/17001474/27938) - would one find this list of valid values?